server/agents/crew_system.py

```
@@ -1,247 +1,258 @@
-        allow_delegation=False,
-        **_llm_kwargs()
-    )
+def _make_llm(agent_name: str):
+    """Build an LLM instance for a specific agent using its dedicated key."""
+    key = _AGENT_KEYS.get(agent_name)
+    if not key:
+        return None
+    try:
+        from crewai import LLM
+        return LLM(model=f"groq/{_MODEL_AGENTS}", api_key=key)
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)